spring-boot/admin/src/main/resources/application.yml

```diff
@@ -92,6 +92,26 @@ spring:
     rest:
       base-path: /api
 
+  security:
+    oauth2:
+      client:
+        registration:
+          # the following oauth client configuration is just for development testing purposes, do NOT use in a production setting
+          github:
+            client-id: e188a243cc57ef90f2b5
+            client-secret: a5630e2f873890bdc8366aac0155e8113ece0997
+            scope[0]: user:email
+            scope[1]: read:user
+            clientAuthenticationScheme: header
+            authenticationScheme: header
+            redirect-uri: http://localhost:8080/login/oauth2/code/github
+        provider:
+          github:
+            # Use the "login" attribute of the Github user info as authentication name in Spring Security
+            user-name-attribute: login
+          resource:
+            userInfoUri: https://api.github.com/user
```

Comment:
See comment about hard-coded URL in Java class: could we re-use this parameter?

Reply:
That is a very good point, but when I try to add it, I always get an error.
![image](https://user-images.githubusercontent.com/47450617/185879035-047461ea-7d78-450f-934b-daef86fb5b3d.png)

Reply:
We should have a look at this together,

Reply:
I endup finding the solution for this issue, while working in the #169 😅 So it seems that the problem was the @AllArgsContructor, so this should be @RequiredArgsConstructor